bccsp/factory/factory.go

```diff
@@ -55,6 +55,16 @@ func GetDefault() bccsp.BCCSP {
 }
 
 func initBCCSP(f BCCSPFactory, config *FactoryOpts) (bccsp.BCCSP, error) {
+	// global var init
```

Comment:
[code hygience]

Reply:
yes, it is tbd code parts. @guijunchen , could you please help adding sm2 logic in this code part?